### PR TITLE
Unlock "undefined"

### PR DESCRIPTION
**Pull request by EndymionJkb**
Changes proposed in this pull request:
- Fix "Unlock undefined to continue" error in Add Liquidity
- Display HTML properly in warning

### CODE REVIEW

File: src/components/Modal/AddLiquidity.vue [hunk 1]
```diff
@@ -306,17 +307,24 @@ export default {
...
-            this.amounts[token],
-            this.web3.tokenMetadata[token].decimals
+            this.amounts[tokenAddress],
+            this.web3.tokenMetadata[tokenAddress].decimals
           )
         ) {
-          return `${this.$t('unlock')} ${token.symbol} ${this.$t(
+          const displaySymbol =
+            typeof token.symbol === 'undefined'
+              ? shortenAddress(tokenAddress)
```
Comment (bonustrack) [inline src/components/Modal/AddLiquidity.vue:324]: You can get the same method from the mixins like this:
```suggestion
              ? this._shortenAddress(tokenAddress)
```

Comment (EndymionJkb) [reply src/components/Modal/AddLiquidity.vue:324]: Ok; updated. I don't entirely understand my own fix (why we have two token lists and this one doesn't have the symbol), but it seems to work

File: src/components/Modal/AddLiquidity.vue [hunk 2]
```diff
@@ -164,7 +164,8 @@ import {
   isTxReverted,
   getTokenBySymbol,
   liquidityToggleOptions,
-  isLocked
+  isLocked,
+  shortenAddress
```
Comment (bonustrack) [inline src/components/Modal/AddLiquidity.vue:168]: You don't need to import shortenAddress anymore here
```suggestion
```

**Review by bonustrack — APPROVED** [review]
Works well

**Pull request merged**